### PR TITLE
Fix deploy and add option to update app

### PR DESCRIPTION
**Pull request by marceloschmidt**

### CODE REVIEW

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -65,26 +66,31 @@ export default class Deploy extends Command {
...
     // tslint:disable:promise-function-async
-    private async asyncSubmitData(data: FormData, flags: { [key: string]: any }): Promise<void> {
+    private async asyncSubmitData(data: FormData, flags: { [key: string]: any }, fd: FolderDetails): Promise<void> {
         const authResult = await fetch(this.normalizeUrl(flags.url, '/api/v1/login'), {
             method: 'POST',
             headers: {
                 'Content-Type': 'application/json',
             },
-            body: JSON.stringify({ username: flags.username, password: flags.username }),
+            body: JSON.stringify({ username: flags.username, password: flags.password }),
```
Comment (graywolf336) [inline src/commands/deploy.ts]: :see_no_evil: whoopies

Comment (marceloschmidt) [reply src/commands/deploy.ts]: Damn!! That's why I don't code anymore 😂

Comment (marceloschmidt) [reply src/commands/deploy.ts]: LOL

Comment (graywolf336) [reply src/commands/deploy.ts]: No no, I was talking about myself :stuck_out_tongue_winking_eye:

Comment (marceloschmidt) [reply src/commands/deploy.ts]: Yeah, I realized after :P then I deleted... hahahha

Comment (sampaiodiego) [reply src/commands/deploy.ts]: LOL

**Pull request merged**